### DISCUSSION TITLE
Question about ID3v2

### DISCUSSION THREAD

**Discussion by FriederHannenheim** [Serial-ATA/lofty-rs#224] [Q&A] [answered]
I looked at a ID3v2 file tagged with lofty-rs in a hex editor and while I did find the values I entered for Artist, Album and Song Title. I didn't find the ID3 header, nor did I find the names of the frames. For some reason the file did show the right tags in easytag, but the track total seemed to be gone. Why is there no ID3 header, and how are the frames encoded?

**Comment by Serial-ATA** [comment, marked as answer]
If you're missing a track total you could actually be saving an ID3v1 tag. Lofty doesn't do any special encoding, it's all spec compliant.

If you're working with `Tag` you can see the type of tag you're working with with `Tag::tag_type()`.

**Comment by FriederHannenheim** [comment]
I just realized my mistake. I was using a file which already had a id3v1 tag by yt-dlp. My bad. Thank you for answering.
